conf/pxe_cluster/hosts.yaml

```diff
@@ -50,6 +50,8 @@ PROVISION:
     ftp_proxy: ""
     http_proxy: "http://10.197.121.21:3142"
     https_proxy: ""
+    ngcacher_proxy: "http://gur44646:Aricent1230@165.225.104.34:80"
+
   PXE:
     password: cable123
```

Comment:
Should consider replacing the real values with something that shows the user they need to change it. Either that or make this entire file a Jinja2 template